src/main/java/com/google/error_analyzer/backend/FilterErrors.java

```diff
@@ -42,6 +45,10 @@
                 documentList.add(document);
             }
         }
+        if (documentList.build().size() == 0) {
+            Document document = new Document("1", 1, NO_ERROR_FOUND_MSG);
```

Comment:
is it necessary?

Reply:
Empty bulk request gives an exception and error index is not created